app/assets/stylesheets/navigation.scss

```diff
@@ -64,10 +64,11 @@ a.button-collapse.top-nav {
       height: 28px;
     }
   }
+}
 
-  .dropdown-content {
-    li {
-      &:hover { background-color: lighten($secondary-color, 50%); }
-    }
+.dropdown-content {
+  background-color: $secondary-color;
+  li > a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3